Admin/Admin.php

```diff
@@ -1620,20 +1653,6 @@ public function setFormGroups(array $formGroups)
     }
 
     /**
-     * {@inheritdoc}
-     */
-    public function removeFieldFromFormGroup($key)
```

Comment:
Methods can be called from an extension, so this is a big BC break. We need to keep them